### ISSUE TITLE
templates should remember start and end time if set

### ISSUE THREAD

**Issue by hferreiro** [Igalia/phpreport#346]
The current behaviour of not setting them would be kept in case the fields aren't filled.

**Comment by anarute** [comment]
@jaragunde I'll start working on this one, can you assign it to me please?

**Comment by jaragunde** [comment]
Merged in PR #505:
```
commit 42c3c4aae1de21b37bec268dacd901e29a2c34a8
Merge: aa4fabd7 2caff4e6
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Thu Aug 5 16:17:56 2021 +0200

    [#346] Save init and end times in the task template
    
    Merge pull request #505 from anarute/feature/i346-save-hours-in-template

commit 7017ed8f796e35f1b01ea8eaa119dec4a2193f2a
Author: Ana Rute Mendes <ana@igalia.com>
Date:   Tue Aug 3 18:21:09 2021 -0300

    Add init and end times to template creation logic

commit e88e21b4df7fd3e93e8ef7ff636d7d7d506c4b88
Author: Ana Rute Mendes <ana@igalia.com>
Date:   Tue Aug 3 17:38:52 2021 -0300

    Add initTime and endTime properties to TemplateVO
    
    The initTimeRaw and endTimeRaw are needed to return the values
    in the format hh:mm right after the template creation. This avoid
    us parsing the time to hh:mm format to int and back to
    hh:mm again

commit 8386510aea8e86c5cc884edb205f69775cbc2ea2
Author: Ana Rute Mendes <ana@igalia.com>
Date:   Tue Aug 3 17:35:48 2021 -0300

    Add init_time and end_time to template table
    
    These columns will be used to stored the task init and end values
```
